nodejs/week3/homework/warmup/index.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express');
+const app = express();
+
+// Parse URL-encoded bodies (as sent by HTML forms)
+app.use(express.urlencoded({ extended: true }));
+// Parse JSON bodies (as sent by API clients)
+app.use(express.json());
+
+app.get('/calculator', (req, res) => {
```

Comment:
Everything seems good, however:
While what you did works, it is too complicated to include everything in a single route. I suggest you create 4 separate routes for the addition/subtraction/multiplication/division. Eg:
```
app.get("/calculator/addition", (req, res)
```

Reply:
oh, okay. Thanks, I did that way in the first, and then I found I have to do this way according to the question. :)